### ISSUE TITLE
please realize analogWrite() function

### ISSUE THREAD

**Issue by djvov** [MajorLee95/nanoI2CIOExpander#3]
Hello, Great library, thank you. But it strongly interesting in analogWrite() function, can you please realize it? )

**Comment by MajorLee95** [comment]
yes great idea. I will do it with eeprom access functions and the possibility to use serial too when i terminate an other great project on Electical Power Strip in line soon (about one month)

**Comment by djvov** [comment]
> yes great idea. I will do it with eeprom access functions and the possibility to use serial too when i terminate an other great project on Electical Power Strip in line soon (about one month)

I will wait!
